TCMSApp/src/client/app/tests/tests-list.html

```diff
@@ -0,0 +1,172 @@
+<div class="matter">
+  <div class="container">
+
+<!-- header -->
+    <div class="clearfix">
+      <h1>
+        Tests
+        <form class="form-inline pull-right">
+          <input type="text" class="form-control" placeholder="Filter">
+        </form>
+      </h1>
+    </div>
+    <hr>
+
+<!--main content-->
+    <div class="row">
+
+<!-- suites -->
+      <div class="col-xs-3">
+        <div class="panel panel-info">
+          <div class="panel-heading">
+            <h3>Suites <button type="button" class="btn btn-primary btn-xs" ng-click="vmTests.openAddSuite()">NEW</button></h3>
+          </div>
+          <div class="table-responsive">
+            <table class="table table-hover" style="font-size:larger">
+              <tbody>
+                <tr ng-repeat="s in vmTests.f track by $index" 
+                    ng-click="seeSuite = !seeSuite; vmTests.setSuite(s)" 
+                    style="cursor: pointer" ng-class="{active: vmTests.compareObj(vmTests.currentSuite, s) }">
+                  <td>{{ ::s.suiteName }}</td>
+                  <td><a ui-sref="run-execute"><i class="fa fa-play-circle fa-lg"></i></a></td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+
+<!-- cases -->
+      <div class="col-sm-9">
+        <div class="panel panel-info">
+          <div class="panel-heading">
+            <h3>Test cases <a ui-sref="tests-create" class="btn btn-primary btn-xs">NEW</a></h3>
+          </div>
+          <div class="table-responsive">
+            <table class="table table-hover">
+              <thead>
+                <tr >
+                  <th>
+                    <input type="checkbox" ng-click = "vmTests.checkedAllCases = !vmTests.checkedAllCases">
+                  </th>
+                  <th>Name</th>
+                  <th>ID</th>
+                  <th>Priority</th>
+                  <th>Created</th>
+                  <th>Automated</th>
+                  <th>Steps</th>
+                  <th>Bugs</th>
+                  <th>Assignee</th>
+                  <th>Run</th>
+                </tr>
+              </thead>
+
+              <tbody ng-repeat="tc in vmTests.currentSuite.tests" style="border:0">
+                <tr ng-click="seeCase = !seeCase;" style="cursor: pointer">
+                  <td ng-click="seeCase = ! seeCase">
```

Comment:
when I'm clicking on checkbox - description for cases (steps, etc) will not open.
So, I wanted to make all area clickable, except checkboxes